### PR TITLE
Add configs to send command

### LINKED ISSUE #8144: Transaction create command fails with send flag for custom data path
### Expected behavior
The command
```
./bin/run transaction create pos stake 100000000 --params='{"stakes":[{"validatorAddress":"lske5sqed53fdcs4m9et28f2k7u9fk6hno9bauday","amount":720000000000}]}' -k "m/44'/134'/1'" -p "my password" --send --data-path ~/myPath
```
should be able to submit transaction to node
### Actual behavior
The command creates a valid transaction but fails to send it to the node. It tries to incorrectly look for a running application in default data path.
```
./bin/run transaction create pos stake 100000000 --params='{"stakes":[{"validatorAddress":"lske5sqed53fdcs4m9et28f2k7u9fk6hno9bauday","amount":720000000000}]}' -k "m/44'/134'/1'" -p "my password" --send --data-path ~/myPath
{
  "transaction": "0a03706f7312057374616b6518002080c2d72f2a200904c986211330582ef5e41ed9a2e7d6730bb7bdc59459a0caaaba55be4ec128321f0a1d0a14b2f75b6968f687b495b69e1fec51d06fa8ae9d8d108080c1b5f4293a40ebf4a106eeed8f27f0a5601cb62cd314d575d8f0508d65a9e8229314448f347b3a315bc7016e6a5203288453ae98c55e7f4946458e2f4598e00a000bb5a68407"
}
{
  "transaction": {
    "module": "pos",
    "command": "stake",
    "fee": "100000000",
    "nonce": "0",
    "senderPublicKey": "0904c986211330582ef5e41ed9a2e7d6730bb7bdc59459a0caaaba55be4ec128",
    "signatures": [
      "ebf4a106eeed8f27f0a5601cb62cd314d575d8f0508d65a9e8229314448f347b3a315bc7016e6a5203288453ae98c55e7f4946458e2f4598e00a000bb5a68407"
    ],
    "params": {
      "stakes": [
        {
          "validatorAddress": "lske5sqed53fdcs4m9et28f2k7u9fk6hno9bauday",
          "amount": "720000000000"
        }
      ]
    },
    "id": "acc3d9aaef1a490cf05443f69ebfcc0ef385c9411caaeb73e698a3932ab3f078"
  }
}
    Error: Application at data path /Users/z/.lisk/pos-mainchain is not running.
```

### Steps to reproduce
Under `example/pos-mainchain` run
```
./bin/run transaction create pos stake 100000000 --params='{"stakes":[{"validatorAddress":"lske5sqed53fdcs4m9et28f2k7u9fk6hno9bauday","amount":720000000000}]}' -k "m/44'/134'/1'" -p "my password" --send --data-path ~/myPath
```

### Which version(s) does this affect? (Environment, OS, etc...)
development

### PR DESCRIPTION
**Pull request by mosmartin**
### What was the problem?

This PR resolves #8144

### How was it solved?

Added the data path and configs to the send command

### How was it tested?

All tests pass

### CODE REVIEW

**Review by shuse2 — COMMENTED** [review]
It looks good but it would be nice to add unit test, also pretty is not considered?

**Comment by mosmartin** [comment]
> It looks good but it would be nice to add unit test, also pretty is not considered?

Pretty? Am not sure I follow 🤔

There is a [test](https://github.com/LiskHQ/lisk-sdk/blob/7d3be6b85c6872fcb313e3f50c65804c881212bd/commander/test/bootstrapping/commands/transaction/create.spec.ts#L626) already for this. Adding more

**Comment by shuse2** [comment]
@mosmartin the send command has flag `--data-path` and `--pretty`, but the logic is handling only the `pretty` path.
Also, the test doesn't check if the flags are passed

**Comment by mosmartin** [comment]
> ... but the logic is handling only the `pretty` path. Also, the test doesn't check if the flags are passed

Based on the issue, the command 👇
`./bin/run transaction create pos stake 100000000 --params='{"stakes":[{"validatorAddress":"lske5sqed53fdcs4m9et28f2k7u9fk6hno9bauday","amount":720000000000}]}' -k "m/44'/134'/1'" -p "my password" --send --data-path ~/myPath` 

Is `--pretty` a default flag?

`--send` invokes the send command which am passing the data path (which was the issue). Should `--pretty` be passed from the above command to the send command?

File: commander/test/bootstrapping/commands/transaction/create.spec.ts [hunk 1]
```diff
@@ -641,6 +641,18 @@ describe('transaction:create command', () => {
 					});
 					expect(stdout[0]).toContain(`Transaction with id: 'undefined' received by node`);
 				});
+
+				it('should invoke send command when --send flag is provided', async () => {
```
Comment (shuse2) [inline commander/test/bootstrapping/commands/transaction/create.spec.ts:645]: it's not checking if the `--data-path` is affecting the send or not

File: commander/test/bootstrapping/commands/transaction/create.spec.ts [hunk 2]
```diff
@@ -624,6 +625,8 @@ describe('transaction:create command', () => {
 			});
 
 			describe('create and send using --send flag', () => {
+				const pathToAppPIDFiles = join(__dirname, 'fake_test_app');
```
Comment (shuse2) [inline commander/test/bootstrapping/commands/transaction/create.spec.ts:628]: it's not wrong, but simply calling `dataPath` seems more straight forward

**Review by shuse2 — APPROVED** [review]

**Review by bobanm — APPROVED** [review]

**Pull request merged**